shared/validation/place-order-request-validator.ts

```diff
@@ -66,7 +67,7 @@ function validateAddress(address: Address, addressType: string): string[] {
     if (address && isNullOrWhitespace(address.postcode)) {
         errors.push(`${addressType} postcode is required`);
     }
-    if (address && !isPostcodeValid(address.postcode)) {
+    if (address.postcode && !isPostcodeValid(address.postcode)) {
```

Comment:
if the postcode not provided by the user, I want to display to him postcode is required
but if he provided the postcode, I want to valid his postcode.
otherwise he will get two error message
1) postcode is required.
2) We don't deliver to your area. However, you can still place an order for collection.